.gitignore

```diff
@@ -2,3 +2,5 @@ build/
 CMakeLists.txt.user
 src/3rd_party-static/openssl-1.0.2/
 *.pyc
+src/components/utils/src/threads/thread.h
```

Comment:
because this file copies in src  folder and need for QT MOC (i.e. for Qt branch only)